src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/Offender.java

```diff
@@ -68,6 +68,7 @@
                 @NamedAttributeNode(value = "sentences", subgraph = "sentence-terms"),
                 @NamedAttributeNode(value = "location"),
                 @NamedAttributeNode(value = "assignedLivingUnit", subgraph = "agency-internal-location-details"),
+                @NamedAttributeNode(value = "releaseDetail"),
```

Comment:
This has to be pulled in as part of the graph, otherwise it attempts to pull it in by its associated bookingId (which can be nullable) - very odd